swagger-java-zenodo-client/src/main/resources/zenodo-1.0.0-swagger-2.0.yaml

```diff
@@ -0,0 +1,815 @@
+swagger: '2.0'
```

Comment:
I guess you can't put it in here, but it would be nice to know, maybe in a commit comment, or maybe in the module's README.md, where this swagger came from.

I thought you said the Zenodo didn't have Swagger doc that you could find? Was this manually created? What if the update our API? How will we know how to update this doc?

Reply:
Zenodo does not have a Swagger doc; it was in Denis' Zenodo prototype code, but we are not sure where it came from. Not a great situation but it works. When I asked Zenodo about it they replied "Thank you for contacting us. Unfortunately, we don't have swagger. You can find a list of the Zenodo REST API at this endpoint: https://zenodo.org/api/
And here you can find the REST API's documentation: http://developers.zenodo.org/"

Reply:
Next sprint https://ucsc-cgl.atlassian.net/browse/SEAB-351